src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -28,6 +30,9 @@
     public static final String MESSAGE_DELETE_EVENT_SUCCESS = "Deleted Event: %1$s";
     public static final String MESSAGE_DELETE_TODO_SUCCESS = "Deleted Todo: %1$s";
 
+    public static final String MESSAGE_UI_PROJECT_NOT_DISPLAYED = "No project displayed. Display a project using the "
+            + ViewProjectCommand.COMMAND_WORD + " command.";
```

Comment:
Was referring to this part, that refers to the COMMAND_WORD. Not a big deal for now since command word will most likely be unchanged.

Reply:
Using COMMAND_WORD allows the CommandWord to be changed easily.